builder-base/install.sh

```diff
@@ -50,7 +50,7 @@ yum install -y \
     unzip \
     wget
 
-GOLANG_VERSION="${GOLANG_VERSION:-1.16.7}"
+GOLANG_VERSION="${GOLANG_VERSION:-1.16.12}"
```

Comment:
youll need to update the checksum files for this one

Reply:
Got it thanks!